src/MentorBot.Functions/App/Startup.cs

```diff
@@ -17,12 +20,16 @@ namespace MentorBot.Functions.App
     public static class Startup
     {
         /// <summary>Registers the services.</summary>
-        internal static void RegisterServices(IServiceCollection services)
+        internal static void RegisterServices(IServiceCollection services, IConfiguration configuration)
         {
             services.AddTransient<IHangoutsChatService, HangoutsChatService>();
             services.AddTransient<ICognitiveService, CognitiveService>();
             services.AddTransient<ICommandProcessor, LocalTimeProcessor>();
             services.AddTransient<IStringLocalizer, StringLocalizer>();
+
+            services.AddTransient<DbContext, ApplicationDbContext>();
```

Comment:
This line appears to be redundant.
You are adding the ApplicationDbContext via the AddDbContextPool helper method.

Reply:
Unfortunately it does not work without `services.AddTransient<DbContext, ApplicationDbContext>();`.